src/containers/App/App.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
This will probably be more integration/acceptance testing once we get to that point. As shown in homepage, these are pretty much the same test, which is probably unnecessary. Totally understand doing this though and thanks!